services/user.py

```diff
@@ -0,0 +1,56 @@
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> QuerySet:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password
+    )
+
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return get_user_model().objects.get(id=user_id)  # чи pk?
+
+
+def update_user(
+        user_id: int,
+        username: str = None,
+        password: str = None,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    user = get_user_model().objects.get(id=user_id)
```

Comment:
The `get` method will raise a `DoesNotExist` exception if no user is found with the given `id`. Consider handling this exception to prevent potential crashes.